### DISCUSSION TITLE
为什么Gemini模型不能收到图片 / Why can't the Gemini model receive images?

### DISCUSSION THREAD

**Discussion by miaotouy** [lobehub/lobehub#2962] [General Question]
![image](https://github.com/lobehub/lobe-chat/assets/66715028/09cdaaa8-97c7-4a1d-b2bc-e3a129f615ae)
我这些都是聚合在new-api里的，Gemini是官方key放在中转里

**Comment by arvinxx** [comment]
建议你问 new-api 。
